metrics/prometheus.go

```diff
@@ -454,6 +455,38 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 			},
 		}...)
 	}
+	if includedMetrics.Has(container.MemoryNumaMetrics) {
+		c.containerMetrics = append(c.containerMetrics, []containerMetric{
+			{
+				name:        "container_memory_numa_pages",
```

Comment:
is this measured in #pages, or bytes?  The help text should specify the units, and the suffix of the metric should be _<units>

Reply:
[Here](https://www.kernel.org/doc/Documentation/cgroup-v1/memory.txt) it is written that memory.numa_stat contains pages and I see that in [runc](https://github.com/opencontainers/runc/blob/d65df61dc5f87e5be4459782ba7a72619f145dca/libcontainer/cgroups/fs/memory.go#L286) values are only read from file. I'll improve help text.